ui/src/ensembler/list/ListEnsemblersView.js

```diff
@@ -63,8 +67,7 @@ export const ListEnsemblersView = ({ projectId, ...props }) => {
     replaceBreadcrumbs([{ text: "Ensemblers" }]);
   }, []);
 
-  const onRowClick = (item) => {};
-  // props.navigate(`./${item.id}/details`);
+  const onRowClick = (item) => { };
```

Comment:
That was added automatically by the default VS Code formatter (since the commented out line was removed) and it passes the lint checks. So I'll leave it as it is.